env.go

```diff
@@ -375,7 +377,10 @@ next:
 				// Fish support was added later. Older hermit envs won't have it.
 				continue next
 			}
-			return errors.Wrapf(err, "%s is missing, not a Hermit environment?", path)
+			return exit.Wrap(
```

Comment:
To test this, I ran `hermit validate env .` locally in a folder without hermit scripts e.g. empty folder, and the see that the exit code `$?` is 82